README.md

```diff
@@ -104,47 +104,91 @@ $ gem install fortnox-api
+
+### Configuration
+The gem is configured in a `configure` block.
+
+Due to Fortnox use of refresh tokens, the gem needs a storage of some sort to keep the tokens. The only thing the storage needs to expose is `access_token` and `refresh_token`. A very simplistic storage would look like this:
 
 ```ruby
-Fortnox::API.configure do |config|
-  config.client_secret = 'P5K5wE3Kun'
-  config.access_token = '3f08d038-f380-4893-94a0a0-8f6e60e67a'
+class MyStorage
+  attr_accessor :access_token
+  attr_accessor :refresh_token
+
+  def initialize(access_token, refresh_token)
+    @access_token = access_token
+    @refresh_token = refresh_token
+  end
 end
 ```
-Before you start using the gem.
 
-### Multiple AccessTokens
+And could then be used like this:
 
-Fortnox uses quite low [API rate limits](https://developer.fortnox.se/blog/important-implementation-of-rate-limits/). The limit is for each access token, and according to Fortnox you can use as many tokens as you like to get around this problem. This gem supports handeling multiple access tokens natively. Just set the `access_tokens` (in plural, compared to `access_token` that only takes a String) to a list of strings:
 ```ruby
 Fortnox::API.configure do |config|
-  config.client_secret = 'P5K5wE3Kun'
-  config.access_tokens ['a78d35hc-j5b1-ga1b-a1h6-h72n74fj5327', 's2b45f67-dh5d-3g5s-2dj5-dku6gn26sh62']
+  config.storage = MyStorage.new(
+    access_token: '[ACCESS_TOKEN]',
+    refresh_token: '[REFRESH_TOKEN]'
+  )
 end
 ```
-The gem will then automatically rotate between these tokens. In theory you can declare as many as you like. Remember that you will need to use one authorization code to get each token! See Fortnox developer documentation for more information about how to get access tokens.
 
-### AccessTokens for multiple Fortnox accounts
-Yes, we support working with several accounts at once as well. Simply set `access_tokens` to a hash where the keys (called a *token store*) represents different fortnox accounts and the value(s) for a specific key is an array or a string with access token(s) linked to that specific Fortnox account. For instance: `{ account1: ['token1', 'token2'], account2: 'token2' }`. If you provide a `:default` token store, this is used as default by all repositories.
+The gem will then automatically refresh the tokens and keep them in the provided storage.
+
+### Support for multiple Fortnox accounts
+Yes, we support working with several accounts at once. Simply expose `token_stores` from the storage and set it to a hash where the keys (called *token store*) represents different Fortnox accounts and provide a refresh token for each such account. If you provide a `:default` token store, this is used as default by all repositories.
 
 ```ruby
+class MyStorage
+  attr_accessor :token_stores
+
+  def initialize(token_stores)
+    @token_stores = token_stores
```

Comment:
You don't need `@token_stores` since you use `attr_accessor`, just `token_stores` is fine (and recommended since it protects you from misspelling the variable name).
So this implementation is completely divorced from the one for a single token? I would expect this to expose a set of `MyStorage` compatible stores. Just for API consistency. Or change the other implementation to act as a hash instead.